src/main/java/com/dev/cinema/controllers/OrderController.java

```diff
@@ -0,0 +1,75 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.dto.OrderRequestDto;
+import com.dev.cinema.dto.OrderResponseDto;
+import com.dev.cinema.dto.TicketsResponseDto;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.OrderService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    @Autowired
+    private UserService userService;
+    @Autowired
+    private ShoppingCartService shoppingCartService;
+    @Autowired
+    private OrderService orderService;
+
+    @PostMapping("/complete")
+    public void complite(@RequestBody OrderRequestDto orderRequestDto) {
```

Comment:
```suggestion
    public void complete(@RequestBody OrderRequestDto orderRequestDto) {
```
крім того, повертати воід - погана практика